src/Integrations/Integrations/Symfony/SymfonyIntegration.php

```diff
@@ -281,14 +284,8 @@ function ($This, $scope) use (&$symfonyCommandsIntegrated) {
             }
         );
 
-        $rootSpan = \DDTrace\root_span();
-        if (null == $rootSpan) {
-            return Integration::NOT_LOADED;
-        }
-
         /** @var SpanData $symfonyRequestSpan */
-        $this->symfonyRequestSpan = $rootSpan;
-        $this->addTraceAnalyticsIfEnabled($rootSpan);
+        $this->symfonyRequestSpan = \DDTrace\root_span();
```

Comment:
What's the purpose of this? Every place but `Symfony\Component\HttpKernel\Event\FilterControllerEvent` and `Symfony\Component\HttpKernel\HttpKernel::handle` do access the actual \DDTrace\root_span(). What is particular about these places?